### PR TITLE
Make API more usable

### PR DESCRIPTION
**Pull request by arantes555**
Hello,

Thanks for your amazing work!
I've used this tool for a small project of mine. However, the library version of it was not very usable as such: it used `process.exit` directly, there was no way of knowing when it was done, no way of specifying where to output the files ... So I made these few small changes, and along the way took this opportunity to clean up the code a bit and bump the dependencies :)

I would love to see this merged so others can use this module as a library!

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+  const pkg = require('./package.json')
   console.log(`Version ${pkg.version}`)
   console.log(`Usage: ${pkg.name} <file.mkv ...>`)
   process.exit(0)
 }
 
-var mkvPaths = process.argv.slice(2)
+const mkvPaths = process.argv.slice(2)
+
+console.log('Processing ' + mkvPaths.join(' & '))
```
Comment (mathiasvr) [inline bin.js]: Is this necessary? The paths are already being outputted below.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (mathiasvr) [inline .gitignore]: I usually don't use .gitignore for common excludes, you can add node_modules to the global ignore file, however if you prefer I don't mind merging it anyway.

**Review by mathiasvr — COMMENTED** [review]
Hi,
Thanks for doing this, there are some nice additions in here. 👍 
I just have a couple of comments but otherwise i'm happy to merge it.

**Comment by arantes555** [comment]
Hi @mathiasvr !
It should be good to merge :)
If there's anything else, I'm happy to tweak it :)

**Comment by mathiasvr** [comment]
Thanks!
I have released it as `v1.1.1`  🎉

**Pull request merged**